src/content-scripts/inject/cosmjs-offline-signer.ts

```diff
@@ -0,0 +1,123 @@
+import {
+  Coin,
+  encodeSecp256k1Signature,
+  OfflineSigner,
+  serializeSignDoc
+} from "@cosmjs/launchpad";
+import { AccountData, SignResponse } from "@cosmjs/launchpad/types/signer";
+import {
+  GetKeyMsg,
+  RequestSignMsg,
+  RequestTxBuilderConfigMsg
+} from "../../background/keyring";
+import { sendMessage } from "../../common/message/send";
+import { BACKGROUND_PORT } from "../../common/message/constant";
+import { StdSignDoc } from "@cosmjs/launchpad/types/encoding";
```

Comment:
This can be imported from top level `@cosmjs/launchpad`

Reply:
Oh, I see. This is a small error in the auto-import feature of IDE.